v21.2/migration-overview.md

```diff
@@ -1,80 +1,87 @@
 ---
-title: Migration Overview
-summary: Learn how to migrate data into a CockroachDB cluster.
+title: Migrate Your Database to CockroachDB
+summary: Learn how to migrate your database to a CockroachDB cluster.
 toc: true
 docs_area: migrate
 ---
 
-CockroachDB supports [importing](import.html) data from the following databases:
+This page summarizes the steps of migrating a database to CockroachDB:
 
-- MySQL
-- Oracle (using CSV)
-- PostgreSQL (and  PostGIS)
+1. [Test and update your schema to work with CockroachDB.](#step-1-test-and-update-your-schema)
+1. [Move your data into CockroachDB.](#step-2-move-your-data-to-cockroachdb)
+1. [Test and update your application.](#step-3-test-and-update-your-application)
 
-and from the following data formats:
+{{site.data.alerts.callout_info}}
+If you need to migrate data from a {{ site.data.products.serverless }} cluster to a {{ site.data.products.dedicated }} cluster, see [Migrate data from Serverless to Dedicated](../cockroachcloud/migrate-from-serverless-to-dedicated.html).
+{{site.data.alerts.end}}
 
-- CSV/TSV
-- Avro
--  ESRI Shapefiles (`.shp`) (using `shp2pgsql`)
--  OpenStreetMap data files (`.pbf`) (using `osm2pgsql`)
--  GeoPackage data files (`.gpkg`) (using `ogr2ogr`)
--  GeoJSON data files (`.geojson`) (using `ogr2ogr`)
+## Step 1. Test and update your schema
 
-This page lists general considerations to be aware of as you plan your migration to CockroachDB.
+To begin a new migration to CockroachDB, extract the [data definition language (DDL)](sql-statements.html#data-definition-statements) of the source database. It is simplest to migrate your database schema to a new CockroachDB database before migrating the data.
```

Comment:
Should we avoid "simplest" here? It's not quite the same as "this is simple to do"... but could invite the same feelings. Is this a recommendation? That is, could you substitute "We recommend migrating your database schema to a new...."

Reply:
Good point; done.